scan/src/pages/HomePage.re

```diff
@@ -2,7 +2,11 @@ module Styles = {
   open Css;
 
   let highlightsContainer =
-    style([width(`percent(100.)), paddingTop(`px(40)), paddingBottom(Spacing.xl)]);
+    style([
```

Comment:
change width 80 to 100 
![image](https://user-images.githubusercontent.com/14128977/71886896-8de5cd00-316f-11ea-8efd-78eaea21ae72.png)